frontends/concrete-python/concrete/fhe/mlir/processors/assign_bit_widths.py

```diff
@@ -65,8 +65,8 @@ def assign_precisions_1_node(node: Node, output_p: int, inputs_p: int):
 CHUNKED_COMPARISON = {"greater", "greater_equal", "less", "less_equal"}
 CHUNKED_COMPARISON_MIN_BITWIDTH = 4
 MAX_POOLS = {"maxpool1d", "maxpool2d", "maxpool3d"}
-MULTIPLY = {"multiply"}
 ROUNDING = {"round_bit_pattern"}
+MULTIPLY = {"multiply", "matmul"}
```

Comment:
This was alphabetically sorted (both variable name and set content). Can you keep it sorted ?

Reply:
I would rather this be done in the future in another PR